### PR TITLE
icecast: depends on libxslt and curl

### PR DESCRIPTION
**Pull request by iggyvolz**
See Homebrew/homebrew#36636.

In trying to install this, I got numerous errors. Fixed by adding a depends_on libxslt and depends_on curl.

These are installed by default on macs, but they must be installed on a non-Mac platform i.e. an Ubuntu with linuxbrew.

### CODE REVIEW

**Comment by sjackman** [comment]
Linuxbrew currently assumes that all software provided by Mac OS is already installed using the host package management system.

**Comment by iggyvolz** [comment]
Is this the "sudo apt-get install build-essential curl..." line on the README?  I thought I did that but I may have forgotten on that particular server.

Could one make these :optional?  The Ubuntu system usually gets out of date and I prefer to use linuxbrew.

**Comment by sjackman** [comment]
Yes, I think I'm going to start including these dependencies as
`depends_on "curl" => :recommended unless OS.mac?`

**Comment by sjackman** [comment]
Could you revise this PR?

File: Library/Formula/icecast.rb [hunk 1]
```diff
@@ -1,5 +1,3 @@
-require 'formula'
-
```
Comment (sjackman) [inline Library/Formula/icecast.rb]: To make merging with Homebrew easier, patches to Linuxbrew should not include unnecessary changes. Changes such as these can be submitted upstream to Homebrew, but they do not accept style-only changes.

Comment (iggyvolz) [inline Library/Formula/icecast.rb]: Whoops - so used to doing that now that it's a matter of instinct.  I'll take that out.

**Comment by iggyvolz** [comment]
How do you want the commit message to be styled for this?  Is "Update icecast dependencies" good or should I change it to the title?

**Comment by sjackman** [comment]
If it fits in 50 characters, then `icecast 2.4.1: Requires libxslt and curl on Linuxbrew`
otherwise `iceast 2.4.1: Add dependencies for Linuxbrew` and list the dependencies in the message.

**Comment by sjackman** [comment]
Thanks for you contribution, Iggy.

**Comment by iggyvolz** [comment]
No problem!  Any time!
